### DISCUSSION TITLE
Add support to Bahrain (BH)

### DISCUSSION THREAD

**Discussion by taljishi** [vacanza/holidays#813] [General]
Hi, 

Is it possible to add support for Bahrain (BH)?

Thank you.

**Comment by arkid15r** [comment]
Hi @taljishi,
I created #814 to track this. 

PH is an open source project so everybody is welcome to contribute. There is no implementation date guarantee though.

Thank you!

**Reply by taljishi** [reply]
Thank you. I will do my part!

**Reply by arkid15r** [reply]
Great! Let me know if you need any help with the implementation.

**Reply by taljishi** [reply]
I'm not a programmer or coder, but I tried my best and created the bahrain.py file. However, not sure how I could run the test to ensure that it meets the standards, and that the holidays are shown properly.

**Reply by arkid15r** [reply]
That's great to hear, @taljishi!
It's okay if the code isn't covered with tests yet. I think we could start from what you have at the moment and improve it further until it meets acceptance criteria. 

You can create a draft PR with bahrain.py and I'll help with the tests and other stuff if needed. Would that work for you?

**Reply by taljishi** [reply]
Thank you @arkid15r! That will work for sure!

I have already created a draft PR with bahrain.py. This is my first time contributing, so I hope I did it right!

**Reply by arkid15r** [reply]
I appreciate your work on the Bahrain holidays. And congrats on the first contribution!!!
I'll do my part soon, perhaps this weekend.

**Comment by arkid15r** [comment]
Bahrain holidays support has been added into beta, and will be included in the next release.
